argparse_test.go

```diff
@@ -511,13 +511,13 @@ func TestFileSimple1(t *testing.T) {
 		t.Errorf("Test %s failed with error: %s", t.Name(), err.Error())
 		return
 	}
-	defer file1.Close()
```

Comment:
I think, this is not good here, if file1 is nil

Reply:
right